install.sh

```diff
@@ -3392,6 +3438,28 @@ setup_shared_hook_repositories() {
     fi
 }
 
+############################################################
+# Sets the githooks templatedir or hookspath
+#   config variable
+#
+# Parameters:
+#   1: path for templateDir or hooksPath
+# 
+# Returns:
+#   none
+############################################################
+
+set_githooks_directory(){
+    if should_use_hooksPath; then
+        mark_as_hookspath_global   
+        git config --global core.hooksPath "$1"
+        git config --global --unset init.templateDir
```

Comment:
I'm not sure we should do this. Someone could have other things in the template directory, it's not only the hooks that come from there.

Reply:
I still don't think we should be unsetting anything here.